### ISSUE TITLE
Unable to start / run on macOS 10.15.7

### ISSUE THREAD

**Issue by RichardMarks** [nrwl/nx-react-native#28]
Hi. I tried to follow the instructions here in the README and it does not work for me.

### What I did

```
npx create-nx-workspace --cli=nx --preset=empty play
cd play
npm install --save-dev @nrwl/react-native
npx nx g @nrwl/react-native:app app
npx nx start app
npx nx run-ios app --install
```

### What Happened

The command `npx nx start app` just returned me to the terminal.
```
$ npx nx start app

> nx run app:start 

$
```

The command `npx run-ios app --install` just returned me to the terminal.
```
$ npx nx run-ios app --install

> nx run app:run-ios --install 

$
```

### What was Expected

I expected the app to run in the ios simulator.

### Version Information

```
$ node -v
v12.18.2

$ npm -v
6.14.5

$ nx --version
11.0.2
```

```json
{
  "@nrwl/react-native": "^10.4.8"
}
```

### System Information

**OS**: macOS Catalina 10.15.7
**Model**: Macbook Pro Retina 15-inch Mid 2015
**Processor**: 2.8 Ghz Quad-Core Intel Core i7
**Memory**: 16GB 1600 Mhz DDR3

### NX Report Information

```
$ nx report

>  NX  Report complete - copy this into the issue template

  Node : 12.18.2
  OS   : darwin x64
  npm  : 6.14.5
  
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.0.2
  @nrwl/cypress : 10.4.4
  @nrwl/devkit : 11.0.2
  @nrwl/eslint-plugin-nx : Not Found
  @nrwl/express : Not Found
  @nrwl/jest : 10.4.4
  @nrwl/linter : 10.4.4
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : Not Found
  @nrwl/react : 10.4.4
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.0.2
  @nrwl/web : 10.4.4
  @nrwl/workspace : 11.0.2
  typescript : 4.0.5
```

**Comment by goofiw** [comment]
Did you get an error when installing `npm install --save-dev @nrwl/react-native`? I had an error in the output.  It wasn't immediately apparent bc the install looks like it succeeds.  I was also able to reproduce this problem.

I worked through this: https://stackoverflow.com/questions/63592445/ld-unsupported-tapi-file-type-tapi-tbd-in-yaml-file .  It seems like it might have something to do with Xcode Command Line Tools and Catalina.

I tried the installing llvm w/ Hombrew, but that hasn't solved the install issue yet (I'm on version 10.1 of xcode to run an old cordova app.  Going to try upgrading XCode.

**Comment by RichardMarks** [comment]
@goofiw 
> Did you get an error when installing `npm install --save-dev @nrwl/react-native`

No, I did not get any errors:

```
$ npm i --save-dev @nrwl/react-native
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated fsevents@1.2.13: fsevents 1 will break on node v14+ and could be using insecure binaries. Upgrade to fsevents 2.

> fsevents@1.2.13 install /Users/username/test-nx-react-native/node_modules/watchpack-chokidar2/node_modules/fsevents
> node install.js

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node

> fsevents@1.2.13 install /Users/username/test-nx-react-native/node_modules/webpack-dev-server/node_modules/fsevents
> node install.js

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node

> core-js@3.8.1 postinstall /Users/username/test-nx-react-native/node_modules/core-js
> node -e "try{require('./postinstall')}catch(e){}"

Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!

The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> https://opencollective.com/core-js 
> https://www.patreon.com/zloirock 

Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)


> core-js-pure@3.8.1 postinstall /Users/username/test-nx-react-native/node_modules/core-js-pure
> node -e "try{require('./postinstall')}catch(e){}"

npm WARN @nrwl/react-native@10.4.8 requires a peer of react-native@^0.63.0 but none is installed. You must install peer dependencies yourself.
npm WARN @nrwl/jest@10.4.4 requires a peer of @nrwl/workspace@10.4.4 but none is installed. You must install peer dependencies yourself.
npm WARN @nrwl/react@10.4.4 requires a peer of @nrwl/workspace@10.4.4 but none is installed. You must install peer dependencies yourself.
npm WARN @nrwl/linter@10.4.4 requires a peer of @nrwl/workspace@10.4.4 but none is installed. You must install peer dependencies yourself.
npm WARN eslint-plugin-react-hooks@2.5.1 requires a peer of eslint@^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN @nrwl/cypress@10.4.4 requires a peer of @nrwl/workspace@10.4.4 but none is installed. You must install peer dependencies yourself.
npm WARN @nrwl/cypress@10.4.4 requires a peer of cypress@>= 3 < 6 but none is installed. You must install peer dependencies yourself.
npm WARN @nrwl/web@10.4.4 requires a peer of @nrwl/workspace@10.4.4 but none is installed. You must install peer dependencies yourself.

+ @nrwl/react-native@10.4.8
added 1423 packages from 562 contributors and audited 1729 packages in 58.523s

103 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by goofiw** [comment]
Hmm, looks like that's a different issue on my end.  I installed (and selected) the new version of xcode and am seeing the same results posted in the initial comment.

**Comment by joewired** [comment]
I had a similar issue. It took me a while to figure it out but for me some modules were missing in the package.json inside the react native app.

something like this:
{
...
"app-module-missing": "*"
..
}

Hope this helps you get to the bottom of it

**Comment by RichardMarks** [comment]
What exactly is missing?

```json
{
  "name": "App",
  "version": "0.0.1",
  "private": true,
  "dependencies": {
    "react-native": "*"
  }
}
```

**Comment by goofiw** [comment]
I (not the original issue owner) got the builder to start working by downgrading nx to version 10.4 (as well as specifying `npx create-nx-workspace@10 play` when making the workspace.

**Comment by jaysoo** [comment]
There was a change with devkit in Nx 11 that caused a breaking change. Just published `11.0.0` of `@nrwl/react-native` to fix the issue.
